guides/release-notes.md

```diff
@@ -7,6 +7,14 @@ We love hearing your feedback! You can help us improve the docs by reporting any
 ![Was this helpful, yes/no buttons](../media/feedback.png)
 
 ---
+### Deprecation Notice: Workflows API
+
+`wix-workflows.v2`
+
+The [Workflows API](https://www.wix.com/velo/reference/wix-workflows-v2) is being discontinued and will no longer be supported moving forward. As this may impact your existing code, we highly encourage you to explore alternative solutions available within Wix to ensure seamless functionality for your projects. 
```

Comment:
Recommend specifying it's the workflows _feature_ that's being discontinued, not just the API.

Reply:
Let's also make it clear that is v2 in the link:
```
  [Workflows API v2](https://www.wix.com/velo/reference/wix-workflows-v2)
```

Reply:
Regarding saying **workflows feature that's being discontinued** need to discuss with Naama (our PM), since we haven't disclosed it to anyone yet. Let's start a thread in #killing-workflows ?